pgen-native/Python.h

```diff
@@ -0,0 +1,56 @@
+#ifndef __PYTHON_H__
+#define __PYTHON_H__
+
+/**
+ * Stub out the bits of python on which which parser.c depends.
+ *
+ */
+
+#include <assert.h>
+#include <ctype.h>
+#include <limits.h>
+#include <stdbool.h>
+#include <stddef.h>
+// XXX: Remove instances of these later
+#define PyAPI_FUNC(x) extern x
+#define PyAPI_DATA(x) extern x
+
+#define PY_SIZE_MAX SSIZE_MAX
```

Comment:
I am pretty sure we could remove these defines and replace them with their definitions as appropriate.